### DISCUSSION TITLE
Salesperson deployment, supports secondary development, assists in resolving bugs(saleor 代部署，支持二开，bug协助解决)

### DISCUSSION THREAD

**Discussion by caohongshan** [saleor/saleor#15770] [Show and tell]
Salesperson deployment, supports secondary development, assists in resolving bugs
I have previously worked on e-commerce second development projects using Saleor and am familiar with the complete process of Saleor. If you need it, please contact QQ: 3161838135

saleor 代部署，支持二开，bug协助解决
之前使用saleor做过电商二开项目，熟悉saleor完整流程，有需要的朋友联系QQ:3161838135

**Comment by aniav** [comment]
I recommend [going to our Discord](https://discord.com/invite/H52JTZAtSH) where we have a job-board channel 🙌
